lang/cs/Org.Apache.REEF.Network.Examples/GroupCommunication/BroadcastReduceDriverAndTasks/BroadcastReduceDriver.cs

```diff
@@ -19,10 +19,8 @@
 using System.Collections.Generic;
```

Comment:
This is an example, right? Do we aim to make the example fault tolerant? If not, I agree with @dkm2110 to leave it as is for now.